test/regression.uts

```diff
@@ -7538,6 +7538,29 @@ assert(p[Ether:2].type == 0x800)
 
 ############
 ############
+############
++ Tests of StreamSocket
+
+= Test with DNS over TCP
+~ netaccess
+
+import socket
+sck = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+sck.connect(("8.8.8.8", 53))
+
+class DNSTCP(Packet):
```

Comment:
We should be able to use the regular `DNS` layer for that, I'll fix that later.

Reply:
I did not find an easy way to achieve the same with `DNS()` without a TCP underlayer.